src/panels/lovelace/entity-rows/hui-media-player-entity-row.ts

```diff
@@ -196,18 +202,20 @@ class HuiMediaPlayerEntityRow extends LitElement implements LovelaceRow {
     `;
   }
 
-  private async _initialLoad(): Promise<void> {
-    await this.updateComplete;
-    this._narrow = false;
-    this._loaded = true;
-
-    if (!this.parentElement) {
+  private _attachObserver(): void {
+    if (typeof ResizeObserver !== "function") {
```

Comment:
let's make this a helper. 

```ts
await ensureResizeObserver()

this._resizeObserver = …
```

Reply:
and share it with other places where we do this.

Reply:
it's not immediately clear to me how to accomplish that. can that be done in a separate PR?

Reply:
Also is this something we want to await? I thought the idea was for it to be async so we don't observer at first for safari but it wouldn't be a big deal